bigquery/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Note this lets me mock final methods, which makes testing google-cloud-java feasible.

Reply:
Ack